java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -115,6 +116,7 @@ public static Object[] transactionTestResult() {
             1L,
             2L,
             value1,
+            2L, // hlen(key4)
```

Comment:
I'm a little bit confused about when we comment these lines or not. I noticed in another PR you did this as well. I thought it was just because a variable was being used instead of a literal, but in this case, it's not consistent with the surrounding literals and variables.

Reply:
The other lines are getting commented in another PR all will be commented.

Reply:
See #160